modules/__tests__/HashRouter-test.js

```diff
@@ -8,6 +8,7 @@ describe('HashRouter', () => {
   const div = document.createElement('div')
 
   afterEach(() => {
+    window.location.hash = ''
```

Comment:
I had to reset this because the `'hashType noslash'` tests are run after the `'hashType hashbang'` tests, at which point the hash is `#!/`, which breaks the `'adds a hash link'` test because it uses a relative URL and would become `#!/foo` instead of `#foo`.